endpoints/auction.go

```diff
@@ -445,16 +440,16 @@ func sortBidsAddKeywordsMobile(bids pbs.PBSBidSlice, pbs_req *pbs.PBSRequest, pr
 				hbSize = width + "x" + height
 			}
 
-			hbPbBidderKey := hbpbConstantKey + "_" + bid.BidderCode
-			hbBidderBidderKey := hbBidderConstantKey + "_" + bid.BidderCode
-			hbCacheIdBidderKey := hbCacheIdConstantKey + "_" + bid.BidderCode
-			hbDealIdBidderKey := hbDealIdConstantKey + "_" + bid.BidderCode
-			hbSizeBidderKey := hbSizeConstantKey + "_" + bid.BidderCode
+			hbPbBidderKey := string(openrtb_ext.HbpbConstantKey) + "_" + bid.BidderCode
+			hbBidderBidderKey := string(openrtb_ext.HbBidderConstantKey) + "_" + bid.BidderCode
+			hbCacheIDBidderKey := string(openrtb_ext.HbCacheKey) + "_" + bid.BidderCode
+			hbDealIDBidderKey := string(openrtb_ext.HbDealIdConstantKey) + "_" + bid.BidderCode
```

Comment:
Do we need the capital 'D' in the new variable names for `hbCacheIdBidderKey` and `hbDealIdBidderKey`?  This sole letter accounts for many changes throughout the document and the change is so little that I find it a bit unnecessary.

Reply:
Honestly, I don't have a very strong opinion on this but I know that this is one of the things that GoLint complains about and it would be great to get this code base in a place where we adhere to GoLint's code style rules which is why I thought I will make this change since I am anyway touching that code

Reply:
Ohh I see. I agree that complying with the GoLint standard is overall a better practice.
Everything else LGTM